utils.go

```diff
@@ -0,0 +1,20 @@
+package check
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/kr/pretty"
+)
+
+// formatUnequal will dump the actual and expected values into a textual
+// representation and return an error message containing a diff.
+func formatUnequal(obtained interface{}, expected interface{}) string {
+	diff := pretty.Diff(obtained, expected)
+	if len(diff) == 0 {
+		diff = []string{fmt.Sprintf("%p != %p", expected, obtained)}
+	}
+
+	return fmt.Sprintf(`Values are different, diff:
+%s`, strings.Join(diff, "\n"))
```

Comment:
The prefix here might be simply "Difference:", and the printing should use the same logic we use in C.logMultiLine so content is properly indented and similar to the rest (see both that and logValue, inside check.go).

Also, this might be inside checkers.go next to the use cases. It's too simple to be on its own file.

Reply:
Thanks, the code uses the logMultiLine logic now and is moved into checkers.go next to where it is used.